easy-deploy-script/easy-deploy-viral-ngs.sh

```diff
@@ -32,10 +32,13 @@ CONDA_ENV_CACHE="conda-cache"
 PROJECTS_DIR="projects"
 MINICONDA_DIR="mc3"
 
-VIRAL_CONDA_ENV_PATH="$SCRIPTPATH/$CONTAINING_DIR/$CONDA_ENV_BASENAME"
-PROJECTS_PATH="$SCRIPTPATH/$CONTAINING_DIR/$PROJECTS_DIR"
-VIRAL_NGS_PATH="$SCRIPTPATH/$CONTAINING_DIR/$VIRAL_NGS_DIR"
-MINICONDA_PATH="$SCRIPTPATH/$CONTAINING_DIR/$MINICONDA_DIR"
+INSTALL_PATH="${VIRAL_NGS_INSTALL_PATH:-$SCRIPTPATH/$CONTAINING_DIR}"
+INSTALL_PATH=$(readlink -f $INSTALL_PATH)
```

Comment:
The OSX `readlink` doesn't have the same meaning for`-f`, so this will fail on OSX, which is why we go through the dance [here](https://github.com/broadinstitute/viral-ngs-deploy/blob/e347f0af33b25290b302ddbfa7b4b5d4acc9f542/easy-deploy-script/easy-deploy-viral-ngs.sh#L11-L21).